### PR TITLE
Images endpoint and unlisted

### PR DESCRIPTION
**Pull request by TianZhiWang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,5 +115,7 @@ Libraries are listed in package.json and requirements.txt
 - https://regex101.com/
 - image tutorial: https://www.youtube.com/watch?v=HSgyPfibrqQ&t=1s
 - http://stackoverflow.com/a/43025574 BKC (https://stackoverflow.com/users/1932097/bkc) MIT
+- get url parameters: https://www.sitepoint.com/get-url-parameters-with-javascript/
+- clipboard js: https://clipboardjs.com/
```
Comment (TianZhiWang) [inline README.md]: Do we really need this library?

File: src/components/PostList.jsx [hunk 2]
```diff
@@ -31,10 +40,75 @@ class PostList extends Component {
...
   loadPosts: PropTypes.func.isRequired,
   posts: PropTypes.array.isRequired,
   toggleFollowStatus: PropTypes.func.isRequired,
   user: PropTypes.object.isRequired,
+  
 };
 
+
+// https://www.sitepoint.com/get-url-parameters-with-javascript/
+function getAllUrlParams(url) {
```
Comment (kylecarlstrom) [inline src/components/PostList.jsx]: Why is this needed @joshdeng ?

Comment (kylecarlstrom) [inline src/components/PostList.jsx]: Seems like this should just be a regex

Comment (kylecarlstrom) [inline src/components/PostList.jsx]: Aren't you just getting the post id from the url. Can't you use a regex like: "([0-9a-zA-Z-]+)\/?$"

File: README.md (hunk 1 above)
Comment (kylecarlstrom) [inline README.md]: I don't think so.

Comment (joshdeng) [inline README.md]: I'm using this for unlisted posts. If you don't need it go ahead delete them.

File: src/components/PostList.jsx (hunk 2 above)
Comment (joshdeng) [inline src/components/PostList.jsx]: For example I can get an object {id:"33262075-83b5-4878-85ce-06d6abff229c"} form url :http://127.0.0.1:8080/?id=33262075-83b5-4878-85ce-06d6abff229c
